content/en/docs/Writing policies/variables.md

```diff
@@ -135,193 +168,28 @@ To refer to values from a ConfigMap inside a `rule`, define a `context` inside t
 
 ```yaml
 rules:
-  - name: example-configmap-lookup
-    # Create a context to invoke a ConfigMap object
+  - name: example-lookup
+    # Define a context for the rule
     context:
-    # Unique name to identify this reference to a ConfigMap
+    # A unique name for the ConfigMap
     - name: dictionary
       configMap:
         # Name of the ConfigMap which will be looked up
-        name: mycmap
+        name: some-config-map
```

Comment:
The name here you changed so it breaks the reference in the example and causes confusion.

Reply:
Agreed. I moved the example back.